options/locale/locale_en-US.ini

```diff
@@ -2883,6 +2883,7 @@ packages.package_manage_panel = Package Management
 packages.total_size = Total Size: %s
 packages.unreferenced_size = Unreferenced Size: %s
 packages.cleanup = Clean up expired data
+packages.cleanup.success = Clean up expired data successfully
```

Comment:
Shouldn't it be `Cleaned up expired data successfully`?

Reply:
My bad.